components/progress/index.zh-CN.md

```diff
@@ -25,6 +25,7 @@ title: Progress
 | percent | 百分比 | number | 0 |  |
 | showInfo | 是否显示进度数值或状态图标 | boolean | true |  |
 | status | 状态，可选：`success` `exception` `normal` `active`(仅限 line) | string | - |  |
+| orientation | 将方向从水平设置为垂直, 可选: `default` `vertical` | string | - |  |
```

Comment:
1. Maybe you forget to update this in docs. `default` => `horizontal`
2. The penultimate grid can be filled in with default values, like 
| options: `horizontal` `vertical` | string | `horizontal` |  3.24.0   |

Reply:
Done